src/hooks/useItems.js

```diff
@@ -0,0 +1,20 @@
+import { useState, useEffect } from 'react';
+import { getItems } from '../services/getItems';
+
+export const useItems = () => {
+  const [items, setItems] = useState([]);
+  const [pageNumber, setPageNumber] = useState(5);
```

Comment:
is there a particular reason your page number starts at 5? usually the page starts at 1?